### DISCUSSION TITLE
Idea: Folders for drafts?

### DISCUSSION THREAD

**Discussion by tommyvillin** [AprilSylph/XKit-Rewritten#1634] [Suggestion Box]
I was thinking it would be super helpful if there could be made an option to add categories or folders to your drafts on Tumblr to sort them into? I have no idea if this is possible to make happen through an extension, but just a thought I had.

**Comment by marcustyphoon** [comment]
Hmm, yeah, that would probably be pretty hard! We can mostly* only modify the content on the page, and Tumblr loads drafts from newest to oldest as you scroll, so we would only have every post in your drafts if you'd scrolled all the way to the end. A folder with only your oldest draft wouldn't be particularly easy to populate.

*it's technically possible to do a lot more than this by, like, entirely replacing the code that displays posts, but this project doesn't have the resources to maintain that much code :D
